pkg/tsdb/cloudwatch/models/cloudwatch_query.go

```diff
@@ -485,3 +490,14 @@ func sortDimensions(dimensions map[string][]string) map[string][]string {
 	}
 	return sortedDimensions
 }
+
+func getEndpoint(region string) string {
+	url := defaultConsoleURL
+	if strings.HasPrefix(region, "us-gov-") {
```

Comment:
rather than pass this a region as an argument, is it possible to pass a partition and to use the aws constants that are exported by the endpoints package? https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants

Reply:
done